modules/govuk_scripts/templates/govuk_node_list_aws.erb

```diff
@@ -153,5 +175,12 @@ parser.add_option(
     dest='classes',
 )
 
+parser.add_option(
+    '--list',
```

Comment:
Maybe `--include-puppet-class` or `--output-puppet-class` would be a more informative option? Given that just running `govuk_node_list` without any arguments will list things.

Reply:
how about `--node-list-with-puppet-class`?

Reply:
Looking at the whole command `govuk_node_list --node-list-with-puppet-class`, the second node list bit seems a bit redundant to me?

Reply:
(but that's definitely more descriptive than `--list` :+1: )